app/views/users/show.html.erb

```diff
@@ -1 +1,42 @@
-<h1>Showing a user profile </h1>
\ No newline at end of file
+<ul class="ul-box">
+  <li class="user-details">
+    <div class="image-box">
+      <%= image_tag(@users.photo, class: "user-image") %>
+    </div>
+    <div class="info-box">
+      <h2><%= @users.name %></h2>
+      <p>Number of posts: <%= @users.post_count %></p>
+    </div>
+  </li>
+  <li class="li-box">
+    <div class="inner-li-box">
+      <h2>Bio</h2>
+      <p class="p_tags"> <%= @users.bio %> </p>
+    </div>
+  </li>
+  <li>
+    <h2 class="single-header">Posts</h2>
+    <% if @users.recent_posts.blank? %>
+      <div class="li-box">
+        <p class="p_tags inner-li-box">There are no posts available.</p>
+      </div>
+    <% else %>
+      <ul class="posts-box">
+        <% @users.recent_posts.each do |post| %>
+          <li class="li-box">
+            <div class="inner-li-box">
+              <h3>Post #<%= post.id %></h3>
+              <p class="p_tags"><%= post.body %></p>
+              <div class="comments_n_likes">
+                <p>Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p>
+              </div>
+            </div>
+          </li>
+        <% end %>
```

Comment:
- [ ] Great Job so far, please note that on the recent posts page (users/show.html.erb) , you should add a link to each post as you did in the view all posts page, kindly update your code where we can access the post from the user show page 

![image](https://github.com/RaphDasilva/blog-app/assets/89282221/a97483bf-856b-4544-ab22-3aaeeaaca542)


when i click on any of this post it should redirect me to that post 🙏